dotcom-rendering/src/footballTables.ts

```diff
@@ -65,8 +76,22 @@ const parseTable = (feGroup: FEGroup): Result<ParserError, FootballTable> => {
 
 const parseTables = listParse(parseTable);
 
-const parseResult = (result: FETeamResult): Result<ParserError, TeamResult> =>
-	ok({
+const parseResult = (result: FETeamResult): Result<ParserError, TeamResult> => {
+	if (isUndefined(result.foe.score)) {
+		return error({
```

Comment:
Is this optional in the frontend model? Could we filter the result out instead of throwing an error on the whole page